config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-periodics-release-1.5.yaml

```diff
@@ -1,86 +1,155 @@
 
-  - name: periodic-cluster-api-provider-vsphere-conformance-release-1-5
-    labels:
-      preset-dind-enabled: "true"
-      preset-cluster-api-provider-vsphere-e2e-config: "true"
-      preset-cluster-api-provider-vsphere-gcs-creds: "true"
-      preset-kind-volume-mounts: "true"
-    decorate: true
-    rerun_auth_config:
-      github_team_slugs:
-      - org: kubernetes-sigs
-        slug: cluster-api-provider-vsphere-maintainers
-    interval: 24h
-    extra_refs:
-      - org: kubernetes-sigs
-        repo: cluster-api-provider-vsphere
-        base_ref: release-1.5
-        path_alias: sigs.k8s.io/cluster-api-provider-vsphere
-    spec:
-      containers:
-        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230719-72ce785784-1.24
-          command:
-            - runner.sh
-          args:
-            - ./hack/e2e.sh
-          env:
-            - name: GINKGO_FOCUS
-              value: "\\[Conformance\\]"
-          # we need privileged mode in order to do docker in docker
-          securityContext:
-            privileged: true
-            capabilities:
-              add: ["NET_ADMIN"]
-          resources:
-            requests:
-              cpu: "4000m"
-              memory: "6Gi"
-    annotations:
-      testgrid-dashboards: vmware-cluster-api-provider-vsphere, sig-cluster-lifecycle-cluster-api-provider-vsphere
-      testgrid-tab-name: periodic-conformance-release-1-5
-      testgrid-alert-email: sig-cluster-lifecycle-cluster-api-vsphere-alerts@kubernetes.io
-      testgrid-num-failures-to-alert: "4"
-      description: Runs conformance tests for CAPV
+- name: periodic-cluster-api-provider-vsphere-test-integration-release-1-5
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  interval: 1h
+  decorate: true
+  rerun_auth_config:
+    github_team_slugs:
+    - org: kubernetes-sigs
+      slug: cluster-api-provider-vsphere-maintainers
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api-provider-vsphere
+    base_ref: release-1.5
+    path_alias: sigs.k8s.io/cluster-api-provider-vsphere
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230719-72ce785784-1.24
+      # we need privileged mode in order to do docker in docker
+      securityContext:
+        privileged: true
+        capabilities:
+          add: [ "NET_ADMIN" ]
+      resources:
+        requests:
+          cpu: "4000m"
+          memory: "6Gi"
+      command:
+      - runner.sh
+      args:
+      - make
+      - test-integration
+  annotations:
+    testgrid-dashboards: vmware-cluster-api-provider-vsphere, sig-cluster-lifecycle-cluster-api-provider-vsphere
+    testgrid-tab-name: periodic-test-integration-release-1-5
+    testgrid-alert-email: sig-cluster-lifecycle-cluster-api-vsphere-alerts@kubernetes.io
+    testgrid-num-failures-to-alert: "4"
+    description: Runs integration tests
+
+- name: periodic-cluster-api-provider-vsphere-e2e-full-release-1-5
```

Comment:
Renamed this from e2e to e2e-full so e2e-full in presubmit and on periodics is the same

e2e without suffix only exists for PRs as this is the only case where we want to run the "PR blocking" subset of e2e tests